statistics/cmsketch_test.go

```diff
@@ -112,10 +113,16 @@ func (s *testStatisticsSuite) TestCMSketch(c *C) {
 }
 
 func (s *testStatisticsSuite) TestCMSketchCoding(c *C) {
-	lSketch, _, err := buildCMSketchAndMap(8, 2048, 0, 1000, 1000, 1.1)
```

Comment:
Why not use this function ?

Reply:
Because I need to construct the worst case, which all values are maximum.